src/components/WorkerInfoModal/WorkerInfoModal.tsx

```diff
@@ -11,8 +11,12 @@ interface WorkerProps {
   open: boolean;
 }
 export const WorkerInfoModal = ({ workerInfo, onClose, open }: WorkerProps) => {
-  const age: number =
-    new Date().getFullYear() - workerInfo.birthDate.getFullYear();
+  let age: number | null;
```

Comment:
why let and why use try and catch? We don't use try and catch in the code almost universally.